test/sparsedir/sparse.jl

```diff
@@ -303,7 +310,8 @@ mfe22 = eye(Float64, 2)
 @test_throws ArgumentError sparsevec([3,5,7],[0.1,0.0,3.2],4)
 
 # issue #5169
-@test nnz(sparse([1,1],[1,2],[0.0,-0.0])) == 0
+# @test nnz(sparse([1,1],[1,2],[0.0,-0.0])) == 0
+# commented following change to sparse() in #14798, also see #12605, #9928, #9906, #6769
```

Comment:
> This PR seems to have fallen off the radar, I think the only thing we were missing was running some package tests and comparing performance on real use cases.

Thanks for the bump! To clarify, was the request for package testing and benchmarking with e.g. Convex.jl directed at me? Apologies if so, I misinterpreted the request. Also if so, how should I go about package testing? Similarly, what benchmarking with Convex.jl would you like to see? Not being a Convex.jl user, I do not know what would constitute a meaningful benchmark.



> Also for these tests where you've commented them out, I think it would be preferable to change the tested-for value, maybe leaving a comment that points to these issues as being the reason for the value to be what it is after this change.

Good call --- I will rebase, touch those tests up, and update the PR. Thanks again!